meta.json

```diff
@@ -19,4 +21,4 @@
     ]
   },
   "entrypoint": "dist/main"
-}
\ No newline at end of file
+}
```

Comment:
:+1: Hooray, having POSIX-compliant newlines at the end of files! I believe @Rob1in's IDE is configured to put them in from now on, at which point we should have fewer of these spurious diffs.